src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -40,14 +41,10 @@
     <Compile Include="System\Runtime\Serialization\SerializationException.cs" />
     <Compile Include="System\Runtime\Serialization\StreamingContext.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'==''">
+  <ItemGroup Condition="'$(TargetGroup)'=='' Or '$(TargetGroup)'=='uap101aot'">
     <Compile Include="System\Runtime\Serialization\InvalidDataContractException.Serialization.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'==''">
-    <!-- ToDo: Remove this dependency once System.Runtime is published. -->
-    <ProjectReference Include="..\..\System.Runtime\pkg\System.Runtime.pkgproj">
-      <Name>System.Runtime</Name>
-    </ProjectReference>
+  <ItemGroup Condition="'$(TargetGroup)'=='' Or '$(TargetGroup)'=='uap101aot'">
```

Comment:
The forwarded types are now in System.Private.Corelib for both NetCore **and** CoreRT. We package the binary that's cross-compiled with TargetGroup='' for uap10.1 (NetCore). So the first clause covers that. But uap101aot needs a specific TargetGroup, in order to trigger injecting BlockReflectionAttribute. So I need to accommodate that by extending the condition with the second clause.
Does that answer your question?